components/app.js

```diff
@@ -0,0 +1,23 @@
+class App {
+  handleGetGradesError(error) {
+    console.error(error)
+  },
+  handleGetGradesSuccess(grades) {
+    console.log(grades)
+  },
+  constructor() {
+    this.handleGetGradesError(),
+    this.handleGetGradesSuccess()
+  },
+  getGrades() {
+    $.ajax({
+      url: 'https://sgt.lfzprototypes.com/api/grades'
+      method: 'GET'
+    })
+    .error(this.handleGetGradesError())
+    .success(this.handleGetGradesSuccess())
```

Comment:
The `handleGetGradesError` and `handleGetGradesSuccess` methods shouldn't be called here. Those methods _should_ be passed to the `error` / `success` methods to be called later, but you shouldn't call them directly.